include/rnp/rnp.h

```diff
@@ -79,6 +79,17 @@ typedef uint32_t rnp_result_t;
 #define RNP_LOAD_SAVE_PERMISSIVE (1U << 8)
 #define RNP_LOAD_SAVE_SINGLE (1U << 9)
 
+/**
+ * Flags for the rnp_key_remove_signatures
+ */
+
+#define RNP_KEY_SIGNATURE_INVALID (1U << 0)
+#define RNP_KEY_SIGNATURE_UNKNOWN_KEY (1U << 1)
+#define RNP_KEY_SIGNATURE_NON_SELF (1U << 2)
+
+#define RNP_KEY_SIGNATURE_LEAVE (0U)
```

Comment:
I would change this to `KEEP` rather than `LEAVE` if I understand what is doing correctly, it's a little clearer

Reply:
Agree, sounds better.